src/prefect/orion/models/block_schemas.py

```diff
@@ -106,9 +106,14 @@ async def create_block_schema(
         .where(
             db.BlockSchema.checksum == insert_values["checksum"],
         )
+        .order_by(db.BlockSchema.created.desc())
```

Comment:
I think we'll need to add an index on `BlockSchema.created` for this

Reply:
It looks like we already have an index on `BlockSchema.created`: https://github.com/PrefectHQ/prefect/blob/9b07ecbe7a0f9b84a61713192dec61e3820a4825/src/prefect/orion/database/orm_models.py#L795